pkgs/development/python-modules/dm-tree/default.nix

```diff
@@ -0,0 +1,47 @@
+{ autoPatchelfHook
+, buildPythonPackage
+, fetchPypi
+, isPy39
+, lib
+, six
+, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "dm-tree";
+  version = "0.1.6";
+  format = "wheel";
+
+  # At the time of writing (8/19/21), there are releases for 3.6-3.9. Supporting
+  # all of them is a pain, so we focus on 3.9, the current nixpkgs python3
+  # version.
+  disabled = !isPy39;
```

Comment:
```suggestion
```

if there is no concrete error on python38, the only other python we currently support, we don't need this.

Reply:
The wheel is Python 3.9 only unfortunately. It produces an error for other versions.

Reply:
That is unfortunate. Can we build from source?

Reply:
Possibly, I can give it a shot. But it has some messy cmake stuff going on, so it may not be straightforward.

Reply:
EDIT: actually the real problem is that it requires bazel. Building python packages that require bazel is a huge PITA, cf. the tensorflow derivation 😬

Reply:
The good news is that the current master branch seems to have dropped bazel in favor of cmake which should be much easier to build from source. So we should take a stab at that on the next release.

Reply:
All i see is cmake. should be fine.

Along with, `dontUseCmakeConfigure = true;`